package.nls.zh-cn.json

```diff
@@ -51,6 +51,8 @@
   "configuration.SAS.results.html.style": "指定 ODS HTML5 结果的样式。",
   "configuration.SAS.results.html.style.(auto)": "让扩展程序选择与颜色主题最匹配的样式。",
   "configuration.SAS.results.html.style.(server default)": "默认为 SAS 服务器上配置的样式。",
+  "configuration.SAS.results.sideBySide": "Display results to the side of the code",
```

Comment:
Hey @scnwwu . Do you mind providing translations for these items when you have a moment?

Reply:
Actually, nevermind for now. I'll create a separate PR for _just_ updating translations